From 90883dd28faf5a0d0a72a59ef6a835616ce28c8c Mon Sep 17 00:00:00 2001
From: Rui Matos <tiagomatos@gmail.com>
Date: Thu, 6 Oct 2016 16:00:40 +0200
Subject: [PATCH] MetaMonitorManagerKms: stop taking drmModeModeInfo.vrefresh
 directly

As in the previous commit, this value already has mode flags baked in
and we can always get the value we need by computing it ourselves.

https://bugzilla.gnome.org/show_bug.cgi?id=772176
---
 src/backends/native/meta-monitor-manager-kms.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/backends/native/meta-monitor-manager-kms.c b/src/backends/native/meta-monitor-manager-kms.c
index 9e3fad2..94d0b4f 100644
--- a/src/backends/native/meta-monitor-manager-kms.c
+++ b/src/backends/native/meta-monitor-manager-kms.c
@@ -414,9 +414,6 @@ drm_mode_vrefresh (const drmModeModeInfo *mode)
 {
   float refresh = 0.0;
 
-  if (mode->vrefresh > 0.0)
-    return mode->vrefresh;
-
   if (mode->htotal > 0 && mode->vtotal > 0)
     {
       /* Calculate refresh rate in milliHz first for extra precision. */
-- 
2.9.3